definitions/features/foreman_proxy.rb

```diff
@@ -3,19 +3,63 @@ class Features::ForemanProxy < ForemanMaintain::Feature
     label :foreman_proxy
   end
 
-  attr_reader :dhcpd_conf_file, :cert_path
+  attr_reader :dhcpd_conf_file, :cert_path, :request_url,
+              :default_ssl_cert_options
 
   def initialize
     @dhcpd_conf_file = '/etc/dhcp/dhcpd.conf'
     @cert_path = ForemanMaintain.config.foreman_proxy_cert_path
+    @request_url = "https://#{hostname}:9090/dhcp"
```

Comment:
Would it be good to use URI library[1] for URL(s) instead of plain strings?
[1]https://ruby-doc.org/stdlib-2.1.2/libdoc/uri/rdoc/URI/HTTP.html